internal/backend/auth/authloginhttpsvc/web/descope.html

```diff
@@ -31,6 +31,7 @@ <h1>Descope Login</h1>
         const wcElement = document.getElementsByTagName("descope-wc")[0];
 
         const onSuccess = (e) => {
+            document.cookie = `DSR=${e.detail.refreshJwt}; path=/`;
```

Comment:
not sure why you set the cookie manually, but you should at least set a secure cookie, same site only and such, better to let the server set the cookie